tasks/jenkins.js

```diff
@@ -30,6 +30,7 @@ module.exports = function (grunt) {
     'rejectRejFiles',
 
     'run:eslint',
+    'run:prettier',
```

Comment:
thoughts on https://github.com/prettier/eslint-plugin-prettier?

Reply:
ah, i see #16339

Reply:
:+1: Yeah, that was just to keep the PR clean as we squash commit this stuff.